lib/resolveurl/plugins/videovard.py

```diff
@@ -25,7 +25,7 @@
 class VideoVardResolver(ResolveUrl):
     name = "videovard"
     domains = ["videovard.sx"]
-    pattern = r'(?://|\.)(videovard\.sx)/[vef]/([0-9a-zA-Z]+)'
+    pattern = r'(?://|\.)(videovard\.sx)/[ved]/([0-9a-zA-Z]+)'
```

Comment:
Why remove `f`? you could leave it in as `[defv]`

Reply:
i removed it cause its not needed. tested a lot of embeds so not sure why its there in the first place. But i can put it back if you want

Reply:
Doesn't seem like the site ever used an `f` at all. Do you want it back in there?

Reply:
okay we can take out `f` as per your change